GitSavvy.sublime-settings

```diff
@@ -149,7 +149,7 @@
     "pedantic_commit": true,
     "pedantic_commit_ruler": true,
     "pedantic_commit_first_line_length": 50,
-    "pedantic_commit_message_line_length": 80,
```

Comment:
Update comment in docstring above

Reply:
What do you mean exactly with this? Sorry I'm a little bit of a noob.